PULL_REQUEST_TEMPLATE.md

```diff
@@ -28,3 +28,40 @@ We must be able to understand the design of your change from this description. I
 ### Applicable Issues
 
 <!-- Enter any applicable Issues here -->
+
+### Metrics
+
+<!-- What metrics are associated with this code change and what questions can they help us answer? Write "N/A" if not applicable. -->
+
+### Tests
+
+<!-- 
+
+How did you verify that your change has the desired effects?
+- How did you verify that all new functionality works as expected?
+- How did you verify that all changed functionality works as expected?
+- How did you verify that the change has not introduced any regressions?
+
+What unit or integration tests were (or will be) added to help protect against future regressions? 
+For manual testing, be sure to describe in detail the actions you performed (including buttons you clicked, text you typed, commands you ran, etc.), and the results you observed.
+If you chose not to include a specific test, please explain why. 
+
+Write "N/A" if not applicable. 
+
+-->
+
+### Documentation
+
+<!-- Describe the documentation added or improved. Write "N/A" if not applicable. -->
+
+### User Experience Research
+
+#### Hypotheses to validate prior to releasing
```

Comment:
I'm still not convinced that this is useful information to collect at the pull request level of granularity.  The pull request template already feels heavy/cumbersome to fill out, for me.  Probably even more so for new contributors. And are we doing anything with this information after we write it out?  We're not going to have enough time to research hypotheses for every pull request.  And I don't think pull requests are where our UXR findings should live because it's hard to see themes in aggregate. 

At the same time, I acknowledge that you said you would find value in writing out the answers to these questions on your own pull requests. How would you feel about clearly marking these questions as optional?  Optional is different from "N/A" in my opinion.  "N/A" is "there are situations where this might not apply" optional is "do this if you would find it valuable."  If you find value in answering these questions in writing, I don't want to stop you.  But I don't think they should be required.

Reply:
Yeah I hear you. Definitely don't want to be increasing friction for contributors.

Definitely agree with making explicit that these are optional (you make a good point about "N/A" not conveying optionality). I'm gonna play with the wording some more and try scaling back this section quite a bit to see how it feels. 

I'm also open to dropping it from the template entirely. In terms of where best to capture this information and have these discussions, @daviwil brought up a good point and suggested that RFCs are a good candidate. That said, we don't always create an RFC for our work and not every change that would benefit from UXR would justify having an RFC. 

As we get into the swing of things with UXR work I'm sure other ideas for keeping track of UXR hypotheses will come up.